app/views/tasks/_edit_modal.html.slim

```diff
@@ -0,0 +1,21 @@
+.modal-dialog role="document"
+  .modal-content
+    .modal-header
+      button type="button" class="close" data-dismiss="modal" aria-label="Close"
+        span aria-hidden="true" &times;
+      h4#editTask.modal-title Edit Task
+
+      = form_for @task, html: {class: "horizontal-form task-form"} do |f|
```

Comment:
@anjali-sharma form should be submitted via ajax so we need to add remote true (this way we'll be able to display errors in modal)